site/en/reference/cli/concepts/srp.md

```diff
@@ -0,0 +1,299 @@
+   start the client by running `srp client start`. 
+
+1. `srp client host` and `srp client service` provide status about whether
+   the client host and service have been successfully registered on the client node.
+
+1. `srp server host` and `srp server service` provide host and service status
+    on the server node.
+
+1. `srp client service remove` removes a service but retains the service name.
+
+1. `srp client host remove` removes the host and all registered services.
+
+## Example Showing Basic SRP server and client commands
+
+The following steps use basic CLI commands to set up a Thread network, start
+the SRP server and client, verify server status, and remove a service. Sample data
+is used for illustrative purposes.
+
+### Start the SRP server
+
+1. Start the SRP server node:
+  
+   ```
+   > ./output/simulation/bin/ot-cli-ftd 1
+   ```
+
+1. Set up a Thread network, then enable the SRP server by running the `srp server enable` command:
+
+   ```
+   > dataset init new
+   Done
+   > dataset
+   Active Timestamp: 1
+   Channel: 22
+   Channel Mask: 0x07fff800
+   Ext PAN ID: 8d6ed7a05a28fb3b
+   Mesh Local Prefix: fded:5114:8263:1fe1::/64
+   Network Key: 7fcbae4153cc2955c28440c15d4d4219
+   Network Name: OpenThread-f7af
+   PAN ID: 0xf7af
+   PSKc: b658e40f174e3a11be149b302ef07a0f
+   Security Policy: 672, onrc
+   Done
+   > dataset commit active
+   Done
+   > ifconfig up
+   Done
+   > thread start
+   Done
+   > state
+   leader
+   Done
+   > ipaddr
+   fded:5114:8263:1fe1:0:ff:fe00:fc00
+   fded:5114:8263:1fe1:0:ff:fe00:c000
+   fded:5114:8263:1fe1:68bc:ec03:c1ad:9325
+   fe80:0:0:0:a8cd:6e23:df3d:4193
+   Done
+   > srp server enable
+   Done
+   ```
+
+### Start the SRP client
+
+1. Start the SRP Client node:
+
+   ```
+   > ./output/simulation/bin/ot-cli-ftd 2
+   ```
+
+1. Join the Thread network, set the client host name and address, and
+   register a service.
+   
+   ```
+   > dataset networkkey 7fcbae4153cc2955c28440c15d4d4219
+   Done
+   > dataset commit active
+   Done
+   > ifconfig up
+   Done
+   > thread start
+   Done
+   > state
+   child
+   Done
+   > ipaddr
+   fded:5114:8263:1fe1:0:ff:fe00:c001
+   fded:5114:8263:1fe1:44f9:cc06:4a2d:534
+   fe80:0:0:0:38dd:fdf7:5fd:24e
+   Done
+   > srp client host name my-host
+   Done
+   > srp client host address fded:5114:8263:1fe1:44f9:cc06:4a2d:534
+   Done
+   > srp client service add my-service _ipps._tcp 12345
+   Done
+   > srp client autostart enable
+   Done
+   ```
+
+   As shown above, use the `srp client autostart enable` command to enable
+   auto-start mode on the client. The client monitors network data to discover
+   available SRP servers within the Thread network, then the client
+   automatically starts itself. Alternatively, start the client manually by
+   issuing the `srp client start` command, and be sure
+   to include the SRP address and port in the command (see below).
+
+1. If manually starting the client, run the following:
```

Comment:
I'd remove the "Alternatively" sentence from this bullet and just incorporate that into the next one.

That way we don't have to use (see below) etc